etc/lighttpd/lighttpd.conf

```diff
@@ -3,18 +3,18 @@ server.modules = (
 	"mod_alias",
 	"mod_compress",
  	"mod_redirect",
+ 	"mod_fastcgi",
 	"mod_rewrite",
 )
 
-server.document-root        = "/var/www/html"
+server.document-root        = "/var/www/client/html"
```

Comment:
why not have a single `html` root?

Reply:
in other words, you'd have `./html` which would contain both `index.html` as well as `api/index.php`? If that's a bad idea then we shouldn't do it, but it seems confusing that there are two webroots.

Reply:
Initially I went for that but `build` re-creates the `html` folder and `web` was mounting it before `build` was able to do its work. So after `build` was done, whatever I mounted on `html` was removed. An alternative is to wait for all of it to be done and symlink the folders into the document root. It doens't bother me this way and it works just fine but if you wanna change it I don't mind it at all